DEVELOPERS.md

```diff
@@ -199,6 +199,18 @@ just test-py
 # run specific test with usual pytest syntax
 just test-py <path/to/test>::<test name>
 
+# run Playwright functional tests
+just test-functional
+
+# run Playwright functional tests with a visible browser
+# This is useful in combination with script pausing when writing tests:
+# https://playwright.dev/python/docs/api/class-page#page-pause
+#
+# Playwright also has other debugging options configurable
+# via the PWDEBUG environment variables, see:
+# https://playwright.dev/python/docs/debug
+just test-functional --headed
```

Comment:
Might be worth giving both commands that I think developers will likely use e.g. 
1. For watching a specific test `just test-functional <path_to_test> --headed --slowmo=500`. 
2. For debugging a specific test `PWDEBUG=1 just test_functional <path_to_test>`

Reply:
Done.